invenio.cfg

```diff
@@ -246,4 +246,4 @@ OAISERVER_ID_PREFIX = "cds-rdm.com"
 
 SEARCH_INDEX_PREFIX = "cds-rdm-"
 COMMUNITIES_ADMINISTRATION_DISABLED = False
-COMMUNITIES_ALLOW_RESTRICTED = True
+COMMUNITIES_ALLOW_RESTRICTED = False
```

Comment:
Why this change?

Reply:
because as far as I know from the previous CDS we will have the need to have restricted communities, unless I misunderstood, we can chat if needed

Reply:
yes and to achieve that you should leave to `True` no?